v3io/common/helpers.py

```diff
@@ -12,16 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-def url_join(base, *parts):
-    result = base
 
-    if result[0] != "/":
-        result = "/" + base
 
-    for part in parts:
-        if part[0] != "/":
+def url_join(*parts):
+    result = ""
+    slash_suffix = False
+    for part_index, part in enumerate(parts):
+        if part == "":
+            continue
+        # add slash prefix before part if:
+        # 1. slash suffix did not exit in prev part
+        # 2. slash prefix does not exit in this part
+        # 3. part is not the first
```

Comment:
exit -> exist

Need to make it clear that all 3 points need to be true (could be read as just one of them).